package/functions/funcs/clientAvatar.js

```diff
@@ -0,0 +1,26 @@
+const clientAvatar = async (d) => {
+  let code = d.command.code;
+
+  const inside = d.unpack();
+  if (!inside.splits[0]) {
+    inside.splits.shift();
+    inside.splits.unshift(d.client.user.id);
```

Comment:
What this thing do?

Reply:
nice question

Reply:
That if guard is not needed nor required.